README.md

```diff
@@ -283,10 +287,12 @@ Import/export things. Corresponds to [`dc_imex()`](https://deltachat.github.io/a
 
 Check if there is a backup file. Corresponds to [`dc_imex_has_backup()`](https://deltachat.github.io/api/classdc__context__t.html#a052b3b20666162d35b57b34cecf74888).
 
-#### `dc.initiateKeyTransfer()`
+#### `dc.initiateKeyTransfer([callback])`
```

Comment:
Callback is _not_ optional.

Reply:
Callback on dc.open is also not optional than?

Reply:
Correct! :smile:

Reply:
I can fix that in another commit.

Reply:
Brackets should be removed.